tasks/xpack/security/elasticsearch-xpack-activation.yml

```diff
@@ -18,7 +18,7 @@
 - name: Activate ES license (with security authentication)
```

Comment:
The license was fixed a while back however this task is still failing. I believe it is because there isn't a valid certificate. This task will need `validate_certs: no` to make sure that the license can be installed. I saw that you added similiar logic to the actual serverspec tests so that will need to be duplicated here.